package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "filestorage",
+  "version": "0.1.0",
+  "author": "",
+  "description": "File storage",
+  "main": "./lib/filestorage.js",
```

Comment:
I think we should add separate ./filestorage.js file and reexport ./lib/filestoage there to be consistent with how other packages handle this.